tweets/14-09-2021-17-33Tweet-on-WG-Multitenanc.tweet

```diff
@@ -0,0 +1 @@
+WG Multitenancy is looking for new contributors to work on virtual cluster project, hierarchical namespace, and multi-tenancy profiles under the WG Multitenancy. For more information, click here: https://github.com/kubernetes/community/tree/master/wg-multitenancy
```

Comment:
```suggestion
WG Multitenancy is looking for new contributors to work on virtual cluster project, hierarchical namespace, and multi-tenancy profiles. For more information, click here: https://github.com/kubernetes/community/tree/master/wg-multitenancy
```

Reply:
Thanks for reviewing @Debanitrkl :) 
I don't know how to edit this pr since it is automated. Should I edit the issue itself or fork the repo and create a pr from there?

Reply:
Just edit this issue and add /validate comment the PR will be automatically updated

Reply:
It doesn't seem to work

Reply:
Weird, it worked fine for me.

Reply:
Could you just accept the changes here in this PR on my suggestion. I guess that would just work fine.

Reply:
I clicked on 'Approve' under 'Review changes' but it doesn't seem to work either.

Reply:
I think /validate is not working once a pr is generated.

Reply:
Raised an issue regarding that it will be resolved soon